pkg/virtualKubelet/liqoNodeProvider/setup.go

```diff
@@ -101,7 +101,7 @@ func node(cfg *InitConfig) *corev1.Node {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:        cfg.NodeName,
 			Labels:      labels.Merge(lbls, cfg.ExtraLabels),
-			Annotations: cfg.ExtraAnnotations,
+			Annotations: cfg.ExtraAnnotations, // TODO: check that they are not overwritten by annotations patch function
```

Comment:
Is this TODO still relevant?

Reply:
No, I checked it, but forgot to remove the comment